config/initializers/assets.rb

```diff
@@ -2,6 +2,13 @@
 #   licensed under the Affero General Public License version 3 or later.  See
 #   the COPYRIGHT file.
 
+# bootstrap-tags plugin relies on rails-assets-bootstrap gem but we use
+# bootstrap-sass this line makes sure we exclude every asset comming
+# from rails-assets-bootstrap to prevent conflicts with bootstrap-sass
+Rails.configuration.assets.paths.reject! do |path|
+  path.include?("rails-assets-bootstrap") && !path.include?("rails-assets-bootstrap-tags")
+end
```

Comment:
Unhopefully, no. I had the same problem on #6551. Using `require: false` was already proposed as a solution [here](https://github.com/tenex/rails-assets/issues/132) but it didn't help.